tests/zha_devices_list.py

```diff
@@ -5919,4 +5919,94 @@
             },
         },
     },
+    {
+        DEV_SIG_DEV_NO: 102,
+        SIG_MANUFACTURER: "Sinope Technologies",
+        SIG_MODEL: "SW2500ZB",
+        SIG_NODE_DESC: b"\x02@\x80|\x11RR\x00\x00\x00R\x00\x00",
```

Comment:
Is there an easy way to pull node descriptions from a device? 
I copied the above from one of the IKEA devices, but I can't imagine that's correct.

Reply:
Once https://github.com/zigpy/zha/pull/177 is merged, you can use the node descriptor dictionary from the "Signature" tab of the device management UI:

<img width="439" alt="image" src="https://github.com/user-attachments/assets/049915b7-2d6d-4e1f-94de-48a6710d7300">